src/ConfigTrait.php

```diff
@@ -90,11 +90,16 @@ protected function getGrantsConfig(ContainerInterface $container) : array
     {
         $config = $container->get('config')['authentication'] ?? [];
 
-        if (empty($config['grants']) || ! is_array($config['grants'])) {
+        if (empty($config['grants'])) {
             throw new InvalidConfigException(
                 'The grant value is missing in config authentication'
```

Comment:
This should be `"grants"`; otherwise, users might add a `grant` configuration value, and still get the exception. I would also likely add a note that the value must be an array.